### ISSUE TITLE
Transition overrides forced scene heading

### ISSUE THREAD

**Issue by rnkn** [ifrost/afterwriting-labs#86]
Using afterwriting CLI 1.9.0, a forced scene heading ending in `TO:` will be exported as a transition, e.g.
~~~
.THROUGH SCOPE:

The speedboat launches directly into the Earth's sun.

.BACK TO:

Max smirks. He begins disassembling his rifle.

CUT TO:
~~~

Here I think the forced scene heading should take priority over the transition, i.e. I imagine the output should be:
- scene heading
- action
- scene heading
- action
- transition

Thoughts?

**Comment by ifrost** [comment]
Thanks for reporting it! You're totally right - forced heading should take priority here. I'll have a look at this :)

**Comment by ifrost** [comment]
Should be okay now! Fixed in `1.9.1`

**Comment by rnkn** [comment]
Excellent work!
